clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2242,11 +2242,10 @@ public DescribeReplicaLogDirsResult describeReplicaLogDirs(Collection<TopicParti
 
         Map<Integer, Set<TopicPartition>> partitionsByBroker = new HashMap<>();
 
-        for (TopicPartitionReplica replica: replicas) {
-            if (!partitionsByBroker.containsKey(replica.brokerId()))
-                partitionsByBroker.put(replica.brokerId(), new HashSet<>());
-            partitionsByBroker.get(replica.brokerId()).add(new TopicPartition(replica.topic(), replica.partition()));
-        }
+		for (TopicPartitionReplica replica : replicas) {
```

Comment:
Can you please fix the indenting here and below?

Reply:
@ijuma Thanks for the review! Yes.  Of course.  Woops.  Sorry about that.